src/ai/backend/common/bgtask.py

```diff
@@ -218,15 +219,16 @@ async def _pipe_builder(r: Redis) -> Pipeline:
 
         await redis_helper.execute(redis_producer, _pipe_builder)
 
-        task = asyncio.create_task(self._wrapper_task(func, task_id, name))
+        task = asyncio.create_task(self._wrapper_task(func, task_id, name, **kwargs))
         self.ongoing_tasks.add(task)
-        return task_id
+        return str(task_id)
```

Comment:
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/44239739/181438475-25170464-8c5c-4b25-8edd-92098f71d660.png">

change `task_id` into str since rpc cannot return this `task_id` in UUID form